### PR TITLE
Duplicate apps and moving buttons

### PR DESCRIPTION
**Pull request by BIG-RAT**
Resolve issues #39 (duplicate apps) and #40 (buttons disappearing).

### CODE REVIEW

File: PPPC Utility.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -372,7 +372,7 @@
 				CODE_SIGN_IDENTITY = "Mac Developer";
 				CODE_SIGN_STYLE = Automatic;
 				COMBINE_HIDPI_IMAGES = YES;
-				DEVELOPMENT_TEAM = "";
+				DEVELOPMENT_TEAM = PS2F6S478M;
```
Comment (cyrusingraham) [inline PPPC Utility.xcodeproj/project.pbxproj]: The development team should be unset

File: PPPC Utility.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -394,7 +394,7 @@
...
```
Comment (cyrusingraham) [inline PPPC Utility.xcodeproj/project.pbxproj]: The development team should be unset

Comment (BIG-RAT) [reply PPPC Utility.xcodeproj/project.pbxproj]: development team has been unset.

**Comment by hisaac** [comment]
Thanks for the pull request @BIG-RAT. We don't want to have the .DS_Store file in source control. Could you please delete it, and then add a `.DS_Store` entry to the .gitignore file as well?

File: .gitignore [hunk 3]
```diff
@@ -21,6 +21,8 @@ xcuserdata/
 *.moved-aside
 *.xccheckout
 *.xcscmblueprint
+.DS_Store
+**/.DS_Store
```
Comment (mm512) [inline .gitignore]: @BIG-RAT I _think_ it may be enough to add here
`*.DS_Store`
instead of those two entries.

Comment (macblazer) [reply .gitignore]: No need for the * in the front at all.  Simply `.DS_Store` will ignore all .DS_Store files in all folders.

**Comment by watkyn** [comment]
@BIG-RAT I'm seeing the button disappearing issue fixed 👍 and also when adding duplicate applications using the + button. But when I drag and drop applications from Finder, it still allows duplicates, so it's probably just something we need to consolidate so the check happens either way the app is added.

File: Source/View Controllers/TCCProfileViewController.swift [hunk 4]
```diff
@@ -159,7 +159,9 @@ class TCCProfileViewController: NSViewController {
             if response == .OK {
                 panel.urls.forEach {
                     guard let executable = self.model.loadExecutable(url: $0) else { return }
-                    block(executable)
+                    if self.model.selectedExecutables.firstIndex(of: executable) == nil {
```
Comment (watkyn) [inline Source/View Controllers/TCCProfileViewController.swift]: We could extract this into a small function and use it for both checks. It helps with clarity about what the code is doing and also helps if we change the logic later, we won't have to remember to change both places.

    func shouldExecutableBeAdded(_ executable: Executable) -> Bool {
        return self.model.selectedExecutables.firstIndex(of: executable) == nil
    }

**Pull request merged**